src/API/Management/Connections.php

```diff
@@ -1,122 +1,187 @@
 <?php
-
+/**
+ * Connections endpoints for the Management API.
+ *
+ * @package Auth0\SDK\API\Management
+ */
 namespace Auth0\SDK\API\Management;
 
-
-use Auth0\SDK\API\Header\ContentType;
-
-class Connections extends GenericResource 
+/**
+ * Class Connections.
+ * Handles requests to the Connections endpoint of the v2 Management API.
+ *
+ * @package Auth0\SDK\API\Management
+ */
+class Connections extends GenericResource
 {
     /**
-     * @param null|string $strategy
-     * @param null|string|array $fields
-     * @param null|string|array $include_fields
+     * Get all Connections by page.
+     *
+     * @param null|string $strategy        - Connection strategy to retrieve.
+     * @param null|string|array $fields    - Fields to include or exclude from the result, empty to retrieve all fields.
+     * @param null|boolean $include_fields - True to include $fields, false to exclude $fields.
+     * @param null|integer $page           - Page number to get, zero-based.
+     * @param null|integer $per_page       - Number of results to get, null to return the default number.
+     * @param array $add_params            - Additional API parameters, over-written by function params.
+     *
      * @return mixed
+     *
+     * @throws \Exception
+     *
+     * @link https://auth0.com/docs/api/management/v2#!/Connections/get_connections
      */
-    public function getAll($strategy = null, $fields = null, $include_fields = null) 
-    {
-        $request = $this->apiClient->get()
-                    ->connections();
-
-        if ($strategy !== null) 
-        {
-            $request->withParam('strategy', $strategy);
+    public function getAll(
+        $strategy = null,
+        $fields = null,
+        $include_fields = null,
+        $page = null,
+        $per_page = null,
+        $add_params = []
+    ) {
+        // Set additional parameters first so they are over-written by function parameters.
+        $params = is_array($add_params) ? $add_params : [];
```

Comment:
again. Is array? shouldn't `$param` be a dictionary structure?

Reply:
Yes, which is == array in PHP ... no special data structure for that